src/main/java/com/mathworks/ci/MatlabInstallationAxis.java

```diff
@@ -0,0 +1,71 @@
+package com.mathworks.ci;
+
+/**
+ * Copyright 2020 The MathWorks, Inc.
+ *
+ * Describable class for MATLAB Axis that provides a list of configured MATLAB installation for
+ * generating matrix configurations.
+ *
+ */
+
+import hudson.Extension;
+import hudson.matrix.Axis;
+import hudson.matrix.AxisDescriptor;
+import hudson.matrix.MatrixProject;
+import org.kohsuke.stapler.DataBoundConstructor;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class MatlabInstallationAxis extends Axis {
+
+    @DataBoundConstructor
+    public MatlabInstallationAxis(List<String> values) {
+        super(Message.getValue("Axis.matlab.key"), evaluateValues(values));
+    }
+
+    static private List<String> evaluateValues(List<String> values) {
+        // Add default configuration is values are null or not selected.
+        if (values == null || values.isEmpty()) {
+            values = new ArrayList<>(Arrays.asList("default"));
+        }
+        return values;
+    }
+
+    @Extension
+    public static class DescriptorImpl extends AxisDescriptor{
+        private final String useMatlabWarning = Message.getValue("Axis.use.matlab.warning");
+        private final String noInstallationError = Message.getValue("Axis.no.installed.matlab.error");
+
+        @Override
+        public String getDisplayName() {
+            return Message.getValue("Axis.matlab.key");
+        }
+
+        @Override
+        public boolean isInstantiable() {
+            return !checkMatlabInstallationEmpty();
+        }
+
+        public boolean checkUseMATLABVersion(Object it) {
```

Comment:
`MATLAB` -> `Matlab`

There's a few more around the rest of the code as well. You might do a find-replace (watch out for user-facing text though, where it should be MATLAB).

Reply:
Done.